src/java/org/apache/cassandra/utils/FBUtilities.java

```diff
@@ -447,19 +446,19 @@ private static Properties getVersionProperties()
             logger.warn("Unable to load version.properties", e);
             return null;
```

Comment:
This seems to have been changed to null instead of `debug_version` in CASSANDRA-17753. I suspect that was by mistake? Same as you were changing in 4.1? Or is there anything I am missing, @jacek-lewandowski ?

Reply:
Why a mistake?

Reply:
I think it was removed because it was unused 🤷🏻‍♂️

Reply:
Its goal was to help not fail cassandra in Intellij, from what I see in the original ticket. So I do not see how it can be exactly used, but I might be missing something. Anyway, we will see with 5.0+ if anyone complains it is missing :-)